index.html

```diff
@@ -5,8 +5,53 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Weather Report</title>
+    <link rel="stylesheet" href="styles/index.css">
 </head>
-<body>
+<body id="body">
+<img src="" alt="">
+    <div class="city_name">
+        <input type="input" id="city_search" placeholder="Your city">
+    </div>
+    <div class="weather">
+        <div class="weather_details">
+                 <h2 id="weather_city">City</h2>
+        </div>
+        <input name="button" id="button" type="button" value="Update!" />
+       
+       <div class="temperature">
+        <h2 class="temp_title">Temperature</h2>
+        <div id="temp_cont">
+            <span class="wrap_value">
+        <span class="value">+45</span>&deg;F
+            </span>
+            <div class="button_wraper">
+        <button id="button_decrease">-</button>
+        <button id="button_increase">+</button>
+        </div>
```

Comment:
Rather than hard-coding a value for UI elements (which just happens to match the initial value backing the control), I like to leave the initial markup blank, and update the UI on first load from the initial backing values in the JS code. So like here, leave off the 45, then when the document loads, update the UI in the JS to show the initial temperature value. This would protect us from changing the default starting JS value, but having it be out of sync with what the UI is showing.

I would tend to do this for any of the configurable elements (temperature, ground, sky, city).